packages/terra-theme-provider/src/ThemeProvider.jsx

```diff
@@ -18,52 +15,39 @@ const propTypes = {
    * Name of class for specified theme. e.g `cerner-consumer-theme`
    */
   themeName: PropTypes.string,
-  /**
-   * When set to true, applies theme class to HTML element
-   */
-  isGlobalTheme: PropTypes.bool,
-};
-
-const defaultProps = {
-  isGlobalTheme: false,
 };
 
 class ThemeProvider extends React.Component {
```

Comment:
Consider turning this into a functional component and using the useEffect hook.

Reply:
Done here : https://github.com/cerner/terra-framework/pull/1031/commits/3747cbde1dbde3f4495849c9f264965539ed05bf